test/framework/systemtools.py

```diff
@@ -1052,6 +1052,19 @@ def test_locate_solib(self):
                 self.assertEqual(os.path.basename(libc_path), libname)
                 self.assertTrue(os.path.exists(libc_path), "%s should exist" % libname)
 
+    def test_find_library_path(self):
+        """Test find_library_path function (Linux and Darwin only)."""
+        if get_os_type() == LINUX:
+            libname = 'libc.so.6'
```

Comment:
This may bite us in the ass if the system on which the test is run doesn't have `libc.so.6`, but I think that's acceptable for now?

Reply:
Ijust copied your test from `locate_libso` :slightly_smiling_face: 
```
    def test_locate_solib(self):
        """Test locate_solib function (Linux only)."""
            if get_os_type() == LINUX:
                libname = 'libc.so.6'
```

Reply:
But I made that test a bit less strict, by allowing that the library is not found. That's not the case here. ;)

Reply:
Let's leave this as is, we can try and come up with a better mechanism if we hit test failures in the future.